app/helpers/date_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module DateHelper
+  def date_range_label(date_range)
```

Comment:
You are not passing a `date_range` you are passing a model should therefore probably be named differently. Maybe `model_with_dates` or something